SMLHelper/Patchers/LanguagePatcher.cs

```diff
@@ -152,13 +155,15 @@ private static bool FileNeedsRewrite(string modKey)
             // Confirm if the file actually needs to be updated
             foreach (string line in lines)
             {
-                string[] split = line.Split(new[] { KeyValueSeparator }, StringSplitOptions.RemoveEmptyEntries);
+                string[] split = line.Split(new[] { KeyValueSeparator }, 2, StringSplitOptions.RemoveEmptyEntries);
 
-                if (split.Length != 2)
-                    return true; // Malformatted, likely externally edited
+                if (split.Length < 2)
```

Comment:
Values greater than `2` would also be invalid if we're trying to keep things to a single `Key:Value` entry per line.

Reply:
Values cannot be greater than 2

Reply:
I know that, and you know that, but for the sake of clarity and self-documentation, we can use `!= SegmentCount` to be very clear about what this code is validating.